haskell-tests/Generators.hs

```diff
@@ -0,0 +1,977 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
It was unnecessary to have this refactoring now.

Either it should have been done sooner, in a separate PR, or later, in a different PR.

This is just adding needless complexity to the review.